README.md

```diff
@@ -6,5 +6,18 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
-1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
-1. Describe the flow of a PATCH request as it goes through the app.
+
+Rails is a framework that is pre-built to serve as a back-end.  We use it as a framework to create the back-end by keeping its structure largely in tact, only changing contents folders like /app/ /config/ and /db/
+We use the framework because we can use it too fit our needs by changing only certain parts of it, but we don't need to rebuild the entire framework from scratch
+
+2. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
```

Comment:
Mostly right; the views are in the wrong place, though. Models do not concern themselves with views, and vice versa. Controllers are the go-between.